src/Tool/Program.cs

```diff
@@ -35,6 +36,19 @@ public static bool TelemetryOptOut
             => true;
 #endif
 
+        /// <summary>
+        /// Creates dictionary of kernelspec file names to the embedded resource path
+        /// of all embedded resources found in `Kernel.csproj`.
```

Comment:
C# doesn't use Markdown for `///`, but uses XML instead. Thus, you'll likely want to modify:
```suggestion
        /// of all embedded resources found in <c>Kernel.csproj</c>.
```